pydantic_numpy/helper/typing.py

```diff
@@ -1,5 +1,10 @@
+from typing import Type
+
+import numpy as np
 from typing_extensions import TypedDict
 
+SupportedDTypes = Type[np.generic]
```

Comment:
```suggestion
SupportedDTypes = type[np.generic]
```

You can also remove the imports of `typing.Type`.

Reply:
great tip! and it should work just fine in Python 3.9.
just did your change.